### PR TITLE
todo-list  best practice

### PR DESCRIPTION
**Pull request by olanikegloria**
In this milestone ;
I am presenting my existing code in a way that allows the student to give a good code review.

### CODE REVIEW

File: src/modules/utility.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  add(list) {
+    this.lists.push(list);
+  }
+
+  remove(itemId) {
+    this.lists = this.lists.filter((item) => item.id !== itemId);
+  }
+}
+
+export const updateLocalStorage = (newListStore) => {
```
Comment (HtetWaiYan7191) [inline src/modules/utility.js]: Kindly consider to separate your updateLocalStorage function from utility.js file . Doing that will make your code more clear and follows best practices. 👍

**Review by HtetWaiYan7191 — CHANGES_REQUESTED** [review]
@HtetWaiYan7191 requested changes on this pull request. 

# STATUS: CHANGES REQUIRED ♻️ 

hello , @olanikegloria , 

Good job so far ! 

 Tasks have required attributes However, there are some issues that you still need to work on to go to the next project but you are almost there ! 👍 

👍 No Linters issue
👍 Good PR 
👍 Good use of Javascript

## Required Changes 

Kindly consider to separate your updateLocalStorage function from utility.js file . Doing that will make your code more clear and follows best practices. 👍 

Cheers and Happy Coding ! 👏 👏 

Feel free to leave any questions or comments in the PR thread if something is not 100 % clear. And please, remember to tag me in your question so I can receive the notification

File: src/modules/utility.js (hunk 1 above)
Comment (olanikegloria) [reply src/modules/utility.js]: @HtetWaiYan7191  Thanks for taking note , I would make the changes right away  👍

**Review by HtetWaiYan7191 — APPROVED** [review]
@HtetWaiYan7191 approved this pull request .

# Approved 👏 

### Highlights 🚀 

- The repo looks professional.
- Git/GitHub flow best practices have been followed 
- You have descriptive PR 
- You meet all requested changes by the previous review.

Your project is complete! There is nothing else to say other than...It's time to merge it 💯 

#### Congratulations 🥳

**Pull request merged**